engine/utils/String.cpp

```diff
@@ -283,6 +285,16 @@ String String::FromLong(const long& value)
     return GetFromFormat("%ld", value);
 }
 
+String String::FromSizeT(const size_t& value)
+{
+    return GetFromFormat("%lu", value);
+}
```

Comment:
`FromInt` handles up to 32 bits no problem. `GetFromFormat` exists if you really need for this use-case, but it would be incredibly rare.

Reply:
Removed.